### DISCUSSION TITLE
KERI Implementors call 09-10-2024

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#98] [Meetings]
Zoom

https://us06web.zoom.us/j/81679782107?pwd=cTFxbEtKQVVXSzNGTjNiUG9xVWdSdz09

Reports

KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

Discussion:

**Comment by m00sey** [comment]
KERIpy:

Open:
- https://github.com/WebOfTrust/keripy/pull/854
- https://github.com/WebOfTrust/keripy/pull/857
- https://github.com/WebOfTrust/keripy/pull/858

Closed:
- https://github.com/WebOfTrust/keripy/pull/859
- https://github.com/WebOfTrust/keripy/pull/860

**Reply by kentbull** [reply]
[#858](https://github.com/WebOfTrust/keripy/pull/858) and [#857](https://github.com/WebOfTrust/keripy/pull/857) are for the same thing, just against two different branches, `main`, and `v1.1.18` respectively.

They fix a couple of things:
1. For the multisig group lead it updates `smids` and `rmids` when creating a multisig rotation.
2. Updtes the `multisig join` and `multisig rotate` commands to send in the `smids` and `rmids`
3. Fixes some of the unit tests to send in `smids` and `rmids`.
4. Adds a unit test that proves that `smids` and `rmids` are accurate for the multisig group lead after creating a rotation.

**Comment by iFergal** [comment]
I'm wondering how we can run watchers and their current state. I've been inspecting watching.py (`Adjudicator`) and the kli commands but not actually sure how to run keripy in watcher mode and how the updates from witnesses are received continuously.

Also, if curious what's involved in building a TEL watcher. I have a use case to verify untargeted ACDC issuances that are publicly available by GET endpoint and IPEX feels overkill.

**Comment by kentbull** [comment]
### Signify-TS

- bugfix: https://github.com/WebOfTrust/signify-ts/pull/273
- two small PRs:
  - https://github.com/WebOfTrust/signify-ts/pull/275
  - https://github.com/WebOfTrust/signify-ts/pull/276

Big discussion on:
- https://github.com/WebOfTrust/signify-ts/issues/274

### CESRide

Nothing new yet.

No new items in the last two weeks.

**Reply by iFergal** [reply]
copying this q from discord relevant to 273

> Question on the rp field - what should this be for /multisig/* messages which have embeds?
>
> It seems like I should send individual messages to each of the other members in a multi-sig (each with the respective rp) but this breaks one of the Signify tests (related to /multisig/rpy notifications for end role auth) so I've a feeling the message should be the same regardless of the recipient.

**Comment by henkvancann** [comment]
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
There is nothing new; we are mainly busy getting the terminology engine (Spec-Up-T) going at ToIP. Which is indirectly relevant for the KERI Suite specifications.

**Comment by rodolfomiranda** [comment]
## KERIA
### No new PRs
### New issues
- https://github.com/WebOfTrust/keria/issues/292
- https://github.com/WebOfTrust/keria/issues/291
- https://github.com/WebOfTrust/keria/issues/290
- https://github.com/WebOfTrust/keria/issues/287

**Reply by 2byrds** [reply]
> ## KERIA
> ### No new PRs
> ### New issues
> * [Contact not added correctly after a previous deletion keria#292](https://github.com/WebOfTrust/keria/issues/292)
> * [Contact attributes disappears after re-resolving OOBI and querying key state keria#291](https://github.com/WebOfTrust/keria/issues/291)
> * [Event driven KERIA operations keria#290](https://github.com/WebOfTrust/keria/issues/290)
> * [Encryption in the KERIA Signify API (and lack of signatures in places currently) keria#287](https://github.com/WebOfTrust/keria/issues/287)

Also would like to discuss the path towards a new KERIA version.
For timing purposes GLEIF may fork KERIA and create intermediate releases (dev, patch, release candidate, and stable). But will continue to contribute back to WebOfTrust so that the community can benefit and release on its own timing.

**Reply by iFergal** [reply]
https://github.com/WebOfTrust/keria/pull/286
https://github.com/WebOfTrust/keria/pull/288
https://github.com/WebOfTrust/keria/pull/289

were merged since last call. though were mentioned last week in ToIP call

**Comment by pfeairheller** [comment]
### SignifyPy

* No new issues or PRs
